plugin.json

```diff
@@ -141,5 +141,8 @@
                 "default": 0
             }
         ]
+    },
+    "props": {
+        "support_packet": "Demo plugin support packet"
```

Comment:
At some point, we should make this a proper field in the manifest. For now, this is fine.

Reply:
Yeah, if we have it's own field I think the plugin won't be backwards compatible hence I went with this.

Reply:
What part of the code would suffer form from a backwards compatible issue?

Reply:
I mean the schema of the manifest needs to be updated so it whomever going to parse the new manifest would require a newer version of the code. Haven't tested though..